File: sandbox-runner/patterns/early-return/java/src/main/java/Shared.java
public final class Shared {
    public static final String TASK_QUEUE = "early-return-task-queue";
    public static final String WORKFLOW_ID_PREFIX = "transaction";

    public record TransactionRequest(double amount, String currency) {}

    public record Transaction(String id, String status) {}

    private Shared() {}
}

File: sandbox-runner/patterns/early-return/go/shared.go
package main

const (
	TaskQueue        = "early-return-task-queue"
	WorkflowIDPrefix = "transaction"
	UpdateName       = "returnInitResult"
)

type TransactionRequest struct {
	Amount   float64 `json:"amount"`
	Currency string  `json:"currency"`
}

type Transaction struct {
	ID     string `json:"id"`
	Status string `json:"status"`
}
